src/parse/parser.rs

```diff
@@ -942,8 +932,11 @@ impl<'help, 'app> Parser<'help, 'app> {
                             .to_string(),
                     ));
                 }
+                .clone();
 
-                bin_name = format!("{} {}", bin_name, &sc.name);
+                sc._build();
+                bin_name.push(' ');
+                bin_name.push_str(&sc.name);
```

Comment:
It looks like the old logic skipped appending `sc.name` for the last entry.    Am I missing something?